crates/pallet-domains/src/tests.rs

```diff
@@ -622,14 +619,13 @@ fn test_bundle_fromat_verification() {
     new_test_ext().execute_with(|| {
         let domain_id = DomainId::new(0);
         let max_extrincis_count = 10;
```

Comment:
Optional: we could fix this typo while we're here:
```suggestion
        let max_extrinsics_count = 10;
```

Reply:
The test name also looks like a typo `test_bundle_fromat_verification`